sensor/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/calibrators/calibrator.py

```diff
@@ -71,6 +71,7 @@ class Calibrator(ParameterizedClass, QObject):
         object, float, int, int, int, float, float, int, int, float, float
     )
 
+    # cSpell:enableCompoundWords
```

Comment:
What is this for?

Reply:
I forgot to delete it after trying to use it for something else